conformity/validator.py

```diff
@@ -131,7 +131,7 @@ def decorated(*passed_args, **passed_kwargs):
 def validate_method(
     kwargs,  # type: Union[fields.Dictionary, fields.SchemalessDictionary]
     returns,  # type: fields.Base
-    args=None,  # type: Union[fields.Tuple, fields.List]
+    args=None,  # type: Optional[Union[fields.Tuple, fields.List]]
```

Comment:
As this line (and some others) already had a type assigned to it just incorrectly not defined as optional I figured it was better to add the optional wrapper instead of replacing the entire type with `type: ignore`